src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,213 @@
+        System.out.println("\t\t" + item + "\n\tNow you have " + (size - 1) + " tasks left!" + LINE_BREAK);
+    }
+
+    /**
+     * Method for message when deleting task for GUI.
+     *
+     * @param item item to be deleted.
+     * @param size size of list with the task included.
+     * @return String message for GUI.
+     */
+    public String guiDeleteMessage(Task item, int size) {
+        return "\t 1 less task! :)\n"
+                + "\t\t" + item + "\n\tNow you have " + (size - 1) + " tasks left!\n";
+    }
+
+    /**
+     * Method for message when marking task as done.
+     *
+     * @param item item to be marked as done.
+     */
+    public void markMessage(Task item) {
+        System.out.println(LINE_BREAK + "\t Great job completing your task! :)");
+        System.out.println("\t\t" + item + LINE_BREAK);
+    }
+
+    /**
+     * Return mark message for GUI.
+     * @param item
+     * @return
+     */
+    public String guiMarkMessage(Task item) {
+        return "\t Great job completing your task! :)\n\t\t" + item;
+    }
+
+    /**
+     * Method for message when marking task as undone.
+     *
+     * @param item to be unmarked.
+     */
+    public void unmarkMessage(Task item) {
+        System.out.println(LINE_BREAK + "\t I see you want to redo this task...");
+        System.out.println("\t\t" + item + LINE_BREAK);
+    }
+
+    /**
+     * Return mark message for GUI.
+     * @param item
+     * @return
+     */
+    public String guiUnmarkMessage(Task item) {
+        return "\t I see you want to redo this task...\n\t\t" + item;
+    }
+
+    /**
+     * Method for message when user asks for whole list.
+     *
+     * @param taskList list to be printed.
+     */
+    public void print(TaskList taskList) {
+        System.out.println(LINE_BREAK + "\tHere are all your tasks, good luck!");
+        System.out.print(taskList.toString());
+        System.out.println(LINE_BREAK);
+    }
+
+    /**
+     * Returns tasklist in string representation for GUI.
+     *
+     * @param taskList tasklist to print.
+     * @return String representation of tasklist.
+     */
+    public String guiPrint(TaskList taskList) {
+        return "\tHere are all your tasks, good luck!\n" + taskList.toString();
+    }
+
+    /**
+     * Method to show user a failed search.
+     */
+    public void failedSearch() {
+        System.out.println(LINE_BREAK + "\tThere are no matching tasks in your list :(\n"
+                + LINE_BREAK);
+    }
+
+    /**
+     * Method to show user a failed search for GUI.
+     *
+     * @return String message for GUI.
+     */
+    public String guiFailedSearch() {
+        return "\tThere are no matching tasks in your list :(\n";
+    }
+
+    /**
+     * Method to print results of a word search.
+     * @param temp taskList to be printed.
+     */
+    public void printSearch(TaskList temp) {
+        System.out.println(LINE_BREAK + "\tHere are the matching tasks!");
```

Comment:
Question: is there a specific reason why you did not combine these 3 lines into one println statement?